src/parser/deserializer.ts

```diff
@@ -21,8 +21,6 @@ export interface DeserializationOptions {
   evalFunctions?: boolean;
   /** cache evaluated functions for reuse. */
   cacheFunctions?: boolean;
-  /** use a crc32 code for caching, otherwise use the string of the function. */
-  cacheFunctionsCrc32?: boolean;
```

Comment:
Unfortunately, this is a breaking change so we can't make it. I'm not against the idea of removing crc32, but at very least we need to properly deprecate it and let users know to stop using the option

Reply:
I think it s a bug fix for what its worth, I'll let Thomas fill in the details but I believe he found there was never a crc32 function in the repo. So maybe we could deprecate this flag but leave that it does nothing, as opposed to adding a new dep. 
As an aside I was wondering what's the advantage of making a hash out of the function string as opposed to using the function string itself as the cache key?

Reply:
I updated the description :)

Reply:
Yeah I totally believe it wasn't there, but we've documented it so we should probably warn users that the option isn't really doing anything. Again, totally fine removing any logic related to crc32.

The benefit of using that would be to speed up lookups in tight loops by providing a faster lookup for a smaller string, and reducing the allocation (in particular if the function is _huge_). Interestingly enough we already embed a FNV1a hash implementation (used to be used for objectid) that would probably be just as effective here. Either way, I think it's premature optimization.

Reply:
@mbroadst I just put the property back in the interface with the deprecated tag, and I removed it from the docs.